packages/utilities/etc/utilities.api.md

```diff
@@ -895,11 +895,14 @@ export const safeSetTimeout: (component: React.Component<{}, {}, any>) => (cb: F
 // @public
 export function setBaseUrl(baseUrl: string): void;
 
+// @public
+export function setFocusRects(enabled: boolean, target?: Element): void;

```

Comment:
Minor nit, but in reading this API, it seems like something like this would be more clear to the customer:

```
setFocusVisibility(visible: boolean, target?: Element)
```

What do you think?

Reply:
that sounds good